tests/src/cgeo/geocaching/files/ProgressInputStreamTest.java

```diff
@@ -9,7 +9,7 @@
 public class ProgressInputStreamTest extends TestCase {
 
     public static void testRead() throws Exception {
-        ProgressInputStream stream = new ProgressInputStream(IOUtils.toInputStream("test"));
+        ProgressInputStream stream = new ProgressInputStream(IOUtils.toInputStream("test", (String) null));
```

Comment:
I would prefer that you use an explicit encoding rather than `null` which is the platform default. Even ASCII would be a better choice than `null`.